brightener.cpp

```diff
@@ -20,14 +12,19 @@ int ImageBrightener::BrightenWholeImage() {
     int attenuatedPixelCount = 0;
     for (int x = 0; x < m_inputImage->m_rows; x++) {
         for (int y = 0; y < m_inputImage->m_columns; y++) {
-            if (m_inputImage->GetPixel(x, y) > (255 - 25)) {
-                ++attenuatedPixelCount;
-                m_inputImage->SetPixel(x, y, 255);
-            } else {
-                int pixelIndex = x * m_inputImage->m_columns + y;
-                m_inputImage->SetPixel(x, y, m_inputImage->GetPixel(x, y) + 25);
-            }
+            attenuatedPixelCount += AttenuatePixel(x, y);
         }
     }
     return attenuatedPixelCount;
 }
+
+int ImageBrightener::AttenuatePixel(int x, int y) {
```

Comment:
The name of this function indicates that it will always attenuate. But it attenuates only in some cases.